docs/providers/aws/guide/functions.md

```diff
@@ -357,10 +357,52 @@ AWS Lambda.
 
 To publish Lambda Layers, check out the [Layers](./layers.md) documentation.
 
-## Log Group Resources
+## Logging
+
+### Log Group Resources
 
 By default, the framework will create LogGroups for your Lambdas. This makes it easy to clean up your log groups in the case you remove your service, and make the lambda IAM permissions much more specific and secure.
 
+### API Gateway Logs
```

Comment:
Those are documented here: https://serverless.com/framework/docs/providers/aws/events/apigateway/#logs

So I believe we should expand documentation at https://github.com/serverless/serverless/blob/7abb23edc8dfbe5005ac716aa137330741759929/docs/providers/aws/events/apigateway.md#logs

Reply:
Good catch, i'll move it there